igvc_description/urdf/worlds/autonav.world

```diff
@@ -0,0 +1,554 @@
+<?xml version="1.0" ?>
+<sdf version="1.4">
+  <world name="default">
+    <scene>
+      <sky>
+        <clouds>
+          <speed>12</speed>
+        </clouds>
+      </sky>
+    <ambient>1.0 1.0 1.0 1.0</ambient>
+    <shadows>true</shadows>
+    </scene>
+
```

Comment:
You should specify the default camera coordinates so it points at the Jessii instead of at the origin.
http://sdformat.org/spec?ver=1.6&elem=world#gui_camera